--- /home/venv/lib/python3.8/site-packages/torch_xla/experimental/pjrt.py	2023-04-03 20:15:07.150108814 +0000
+++ /home/venv/lib/python3.8/site-packages/torch_xla/experimental/pjrt_patched.py	2023-04-03 20:17:14.973979564 +0000
@@ -268,7 +268,7 @@
 
 @requires_pjrt
 def _initialize_multiprocess(local_rank: int, local_world_size: int):
-  os.environ.setdefault('LOCAL_RANK', str(local_rank))
+  #os.environ.setdefault('LOCAL_RANK', str(local_rank))
   os.environ.setdefault('LOCAL_WORLD_SIZE', str(local_world_size))
 
   if device_type() == 'TPU':
@@ -488,3 +488,4 @@
         dist.broadcast(b, src=0)
 
     return self._module(x)
+
